templates/credentials/beta-activation.html

```diff
@@ -1,40 +1,43 @@
 {% extends "credentials/base_cred.html" %}
 
-{% block title %}Canonical Credentials -- Beta activation{% endblock %}
+{% block title %}Beta activation | Credentials{% endblock %}
 
 {% block meta_description %}
-    The Canonical Ubuntu Essentials exams certify knowledge and verify skills in general Linux, Ubuntu Desktop, and Ubuntu Server topics.
+  The Canonical Ubuntu Essentials exams certify knowledge and verify skills in general Linux, Ubuntu Desktop, and Ubuntu Server topics.
 {% endblock meta_description %}
 
 {% block content %}
 
-    <section class="p-strip--suru-topped">
-        <div class="row">
-            <h1>Add activation keys</h1>
-            <div class="col-8">
-                <form action="/credentials/beta/activation" method="post">
-                    <label class="p-heading--3" for="Emails">Email addresses</label>
-                    <p>Add one or more user email addresses below, each on a new line.</p>
-                    <textarea id="Emails"
-                              name="emails"
-                              class="is-required"
-                              required
-                              aria-required="true"
-                              rows="10"
-                              placeholder="charlie.changer@example.com"></textarea>
-                    <label class="p-heading--3" for="Keys">Activation keys</label>
-                    <p>Add one or more activation keys below, each on a new line.</p>
-                    <textarea id="Keys"
-                              name="keys"
-                              class="is-required"
-                              required
-                              aria-required="true"
-                              rows="10"
-                              placeholder="abcdef"></textarea>
-                    <button type="submit" class="p-button--positive">Submit</button>
-                </form>
-            </div>
+  <section class="p-strip is-bordered is-deep">
```

Comment:
Bordered strips are deprecated, and in this case it's not really visible anyway, because of the orange border of the footer.

```suggestion
  <section class="p-strip is-deep">
```